packages/wdio-firefox-profile-service/tests/launcher.test.ts

```diff
@@ -176,5 +178,24 @@ describe('Firefox profile service', () => {
 
             expect(FirefoxProfile.copy).toHaveBeenCalledWith(options.profileDirectory, expect.any(Function))
         })
+
+        test('should not continue if profile could not be created', async () => {
+            const options = {
+                profileDirectory: '/tmp/firefox-profile'
+            }
+            const capabilities: WebDriver.DesiredCapabilities[] = [{
+                browserName : 'firefox',
+            }]
+
+            ;(FirefoxProfile.copy as any as jest.Mock).mockImplementationOnce((profileDirectory: string, cb: Function) => cb())
+            const service = new Launcher(options)
+            await service.onPrepare({}, capabilities)
+        })
```

Comment:
Should these tests have assertions? It looks like the code is being run, but I don't see an assertion for this one or the next test that was added.

Reply:
no assertion needed because the test would fail if we wouldn't return early in the onPrepare hook. Maybe I can make this more clear?